addons/base_import_module/models/base_import_module.py

```diff
@@ -11,7 +11,7 @@ class BaseImportModule(models.TransientModel):
 
     module_file = fields.Binary(string='Module .ZIP file', required=True, attachment=False)
     state = fields.Selection([('init', 'init'), ('done', 'done')], string='Status', readonly=True, default='init')
-    import_message = fields.Char()
+    import_message = fields.Html()
```

Comment:
Why is `Html` field type required for this task? Isn't `Text` field sufficient?

Reply:
Yep, It is. For text field we need to add 'html' widget in the view : https://github.com/odoo/odoo/pull/33029/files#diff-d10ecf0d8385c027c87d7f1e8e7e5fc5R19 seems okay ?

Reply:
Nice :) Mmh, it shouldn't be needed ; `html` widget is the same implementation as a `text` one (see https://github.com/odoo/odoo/blob/master/addons/web/static/src/js/fields/field_registry.js#L30 ) and `text` field widget will be used if the field is a `field.Html`.

Reply:
@mge-odoo I think it's overridden with https://github.com/odoo/odoo/blob/5dc971b1ae15a9f9e6e92914a3b0d98159c9a38f/addons/web_editor/static/src/js/backend/field_html.js#L495 because web_editor is auto install module.

Reply:
Oh you are right about `html`, my bad :)

But my first point was that you should'nt need html at all. Like why do you replace `\n` by `<br/>` in https://github.com/odoo/odoo/pull/33029/files#diff-ae7b12449abf40341cde1280088b36c0R25 ? The `text` field (and corresponding widget) should be able to display the line break correctly.

Reply:
Hello @mge-odoo 
Agree with your point.! Text field and html widget will handle it.
I forgot to remove it.
Could you please check it again ?

Reply:
@jso-odoo seems better now, thanks :)